index.html

```diff
@@ -184,6 +187,42 @@ <h3>
           </tr>
         </tbody>
       </table>
+      <h4>
+        [=Top-level traversable=]'s {{Document}} interface
+      </h4>
+      <p>
+        The following internal slots are added to the
+        [=top-level traversable=]'s {{Document}} interface.
```

Comment:
What we discussed yesterday was that this internal slot belongs to the top-level traversable itself, not to its Document. The previous version of your text was already setting this value only in the top-level traversable's Document, and my point was that instead of using the internal slot only in certain Documents, you should skip the Document part altogether and add this data to the top-level traversable.

Think about it this way too: if you set it to a Document, it will be unset whenever there's a navigation (i.e. you go from one page to another in the tab). If you assign it to the top-level traversable, the override will remain across navigations, which is the behavior we want.

Reply:
Sorry I thought the "document" in here is not other item but place/structure to have the internal slot. (Badly explained)

Next problem is word "interface" . That came from {{Document}}. Word "interface" cannot be used with "top-level traversable" ? These sentences come so weird. So is the header then
"[relevant global object](https://html.spec.whatwg.org/multipage/webappapis.html#concept-relevant-global)'s [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable)'s [top-level traversable](https://html.spec.whatwg.org/multipage/document-sequences.html#top-level-traversable)" OR just `[=Top-level traversable=]`?

Reply:
"Interface" has a very specific meaning: https://webidl.spec.whatwg.org/#idl-interfaces. `Document` is an interface because it is declared as such in https://dom.spec.whatwg.org/#interface-document and further extended in the HTML spec.

A top-level traversable does not have a Web IDL definition and cannot be called an interface.

> So is the header then
"[relevant global object](https://html.spec.whatwg.org/multipage/webappapis.html#concept-relevant-global)'s [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable)'s [top-level traversable](https://html.spec.whatwg.org/multipage/document-sequences.html#top-level-traversable)" OR just [=Top-level traversable=]?

The former. If you just say "the top-level traversable", you need to explain which top-level traversable you are referring to and where you are getting it from, which leads you back to the first version.

Reply:
Fixed in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/d9d848ca4807f9ca82e392f6305b9631daa6272f